src/core/mac/sub_mac.cpp

```diff
@@ -396,6 +396,8 @@ Error SubMac::Send(void)
 {
     Error error = kErrorNone;
 
+    mTransmitFrame.SetTxPower(OT_RADIO_POWER_INVALID);
```

Comment:
How about defining  `static constexpr int8_t kInvalidTxPower = OT_RADIO_POWER_INVALID; ///< Invalid TX power value.` in `radio.hpp` and use it within core?